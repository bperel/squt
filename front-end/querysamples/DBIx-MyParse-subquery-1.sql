SELECT field1 FROM table1 WHERE cond1
