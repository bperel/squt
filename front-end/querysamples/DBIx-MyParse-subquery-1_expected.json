{
	"Tables" : {
		"table1" : {
			"table1" : {
				"OUTPUT" : {
					"field1" : {
						"-1" : "field1"
					}
				},
				"CONDITION" : {
					"cond1" : {
						"EXISTS" : "0"
					}
				}
			}
		}
	},
	"Functions" : {
		"0" : {
			"name" : "EXISTS",
			"to" : "NOWHERE"
		}
	}
}
